components/clicktime/clicktime.app.mjs

```diff
@@ -0,0 +1,192 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "clicktime",
+  propDefinitions: {
+    accountingPackageId: {
+      type: "string",
+      label: "Accounting Package ID",
+      description: "Unique identifier for the accounting package",
+    },
+    billingRate: {
+      type: "string",
+      label: "Billing Rate",
+      description: "The billing rate for the client, job, or user",
```

Comment:
Description should be defined separately on each component to reflect the type of resource.

Please do this for all prop descriptions that are referring to "entity" or "client, job or user".